vnext/JSI/Shared/ChakraRuntime.cpp

```diff
@@ -902,17 +978,16 @@ facebook::jsi::Value ChakraRuntime::HostObjectOwnKeysTrap(
   auto ownKeys = hostObject->getPropertyNames(chakraRuntime);
 
   std::set<std::string> dedupedOwnKeys{};
-  for (size_t i = 0; i < ownKeys.size(); ++i) {
-    dedupedOwnKeys.insert(ownKeys[i].utf8(chakraRuntime));
+  for (const auto &ownKey : ownKeys) {
+    dedupedOwnKeys.insert(ownKey.utf8(chakraRuntime));
   }
 
   size_t numKeys = dedupedOwnKeys.size();
   facebook::jsi::Array result = chakraRuntime.createArray(numKeys);
 
   size_t index = 0;
   for (const std::string &key : dedupedOwnKeys) {
-    result.setValueAtIndex(chakraRuntime, index, facebook::jsi::String::createFromUtf8(chakraRuntime, key));
-    ++index;
+    result.setValueAtIndex(chakraRuntime, index++, facebook::jsi::String::createFromUtf8(chakraRuntime, key));
```

Comment:
>index++ [](start = 42, length = 7)

(nit) it's a neat shorthand, but I prefer explicit separate ++index statement, like before for readability.